frontend/assets/l10n/app_de.json

```diff
@@ -109,6 +109,9 @@
     "codeVerificationFailedConnection": "Der eingescannte Code konnte nicht verifiziert werden. Bitte stellen Sie sicher, dass eine Internetverbindung besteht und prüfen Sie erneut.",
     "compareWithID": "Gleichen Sie die angezeigten Daten mit einem amtlichen Lichtbildausweis ab.",
     "comparedWithID": "Ich habe die Daten mit einem amtlichen Lichtbildausweis abgeglichen.",
+    "extendCard": "Karte verlängern",
+    "extendCardNotificationTitle": "Hinweis zum Ablauf der Karte",
+    "extendCardNotificationDescription": "Ihre Karte läuft demnächst ab.\nVerlängern Sie sie jetzt, um Ihre Dienste weiterhin nutzen zu können.",
```

Comment:
"Dienste" sounds a bit odd to me...

Maybe just "..., um sie weiterhin nutzen zu können." ?

Reply:
what about 'um Ihre Services weiterhin nutzen zu können.' ? 
just thinking if the original meaning could be preserved

Reply:
Maybe, "um Ihre Vorteile weiterhin erhalten/nutzen zu können."? Maybe in English "... to continue receiving your benefits."?

Reply:
But maybe it only sounds odd to me and "Dienste" is also fine.

Reply:
I like benefits/Vorteile